discovery-infra/test_infra/utils/env_var_keys.py

```diff
@@ -0,0 +1,50 @@
+
+class EnvVarsKeys:
+    SSH_PUB_KEY = "SSH_PUB_KEY"
+    REMOTE_SERVICE_URL = "REMOTE_SERVICE_URL"
+    PULL_SECRET = "PULL_SECRET"
+    OFFLINE_TOKEN = "OFFLINE_TOKEN"
+    OPENSHIFT_INSTALL_RELEASE_IMAGE = "OPENSHIFT_INSTALL_RELEASE_IMAGE"
+    OPENSHIFT_VERSION = "OPENSHIFT_VERSION"
+    BASE_DOMAIN = "BASE_DOMAIN"
+    WORKERS_COUNT = "WORKERS_COUNT"
+    MASTERS_COUNT = "MASTERS_COUNT"
+    NUM_MASTERS = MASTERS_COUNT  # For backwards compatibility
```

Comment:
Could you open a ticket to fix it? so we won't need this backwards compatibility.